Modules/SubModule_Structural/xASL_wrp_VisualQC_Structural.m

```diff
@@ -166,6 +166,9 @@ function xASL_wrp_VisualQC_Structural(x)
 end
 
 if  xASL_exist(x.P.Path_T2,'file')
+	% It is created only if the T2 file exists
+	xASL_adm_CreateDir(x.D.T2_CheckDir);
```

Comment:
yes and let's make an issue to move this to the CreateFigure function instead. That makes debugging easier.

Reply:
Created a new issue for that.